recipes/crystmatch/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "crystmatch" %}
+{% set version = "1.1.2" %}
+{% set python_min = "3.9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/crystmatch-{{ version }}.tar.gz
+  sha256: 0e3a87a1582788b96a77b9be2a210bfaa3bd16a05d0701c23f27a8bf56272302
+
+build:
+  entry_points:
+    - crystmatch = crystmatch:main
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python {{ python_min }}
+    - setuptools >=42
+    - wheel
+    - numpy
```

Comment:
Just as a general note, all of the requirements below `wheel` (except for `pip`), are only needed because you put them in the `requires` section under `build-system` in your upstream `pyproject.toml`. Looking at the `setup.py` they do not appear to be strictly required to "build" the package. They are only really runtime dependencies. If I'm correct about this, it might be beneficial in a future release to fix that upstream.

Reply:
You are correct. I removed these lines in `meta.yaml` as well as those in `pyproject.toml`. Thanks!